packages/SwingSet/src/liveslots/vatstore-usage.md

```diff
@@ -63,50 +62,53 @@ In a c-list or virtualized data, you may see vrefs like these:
 * `o+12/2:0`: the first facet of a different instance
 * `o+12/3:0`: the first facet of another different instance
 
-Each instance of a virtual object stores state in a vatstore key indexed by the baseref. If `o+12/1:0` and `o+12/1:1` are the facet vrefs for a cohort whose baseref is `o+12/1`, the cohort's shared state will be stored in `vs.vom.o+12/1`, as a JSON-serialized record. The keys of this record are property names: if the Kind uses `state.prop1`, the record will have a key named `prop1`. For each property, the value is a capdata structure: a record with two properties `body` and `slots`.
+Each instance of a virtual object stores state in a vatstore key indexed by the baseref. If `o+12/1:0` and `o+12/1:1` are the facet vrefs for a cohort whose baseref is `o+12/1`, the cohort's shared state will be stored in `vom.o+12/1` as a JSON-serialized record. The keys of this record are property names: if the Kind uses `state.prop1`, the record will have a key named `prop1`. For each property, the value is a capdata structure: a `{ body: string, slots: any[] }` record.
 
 * `v6.vs.vom.o+12/1` : `{"prop1":{"body":"1","slots":[]}}`
```

Comment:
@warner Are the values associated with Kind state properties actually capdata structures, or are they instead JSON serializations of such records (as is more common throughout our systems)?
```suggestion
* `v6.vs.vom.o+12/1` : `{"prop1":"{\"body\":\"1\",\"slots\":[]}"}`
```